src/data/real/ennreal.lean

```diff
@@ -408,6 +408,43 @@ iff.intro
 
 end sub
 
+section sum
+
+open finset
+
+/-- sum of finte numbers is still finite -/
+lemma sum_lt_top [decidable_eq α] {s : finset α} {f : α → ennreal} :
```

Comment:
This lemma and the next one are not really specific to `ennreal`. They could be stated and proved for any `with_top α` when `[add_semigroup α]`, in `algebra/ordered_group/`. Then you can state the ennreal version in `ennreal.lean`.